src/components/Contacts/ContactListTable.jsx

```diff
@@ -44,31 +50,74 @@ const ContactListTable = ({ contacts, deleteContact }) => {
   const contactsCopy = [...contacts];
   const sortedContacts = contactsCopy.sort(comparePerson);
 
+  const columnTitlesArray = [
+    {
+      field: 'First Name',
+      width: 120,
```

Comment:
Could we possibly expand the table a bit when in desktop views? The quickest way would be by widening the columns.

I know there isn't a ton of data to display at the moment, but it'd help to use up some of all the empty space. Plus, in case anyone has a longer name, it'd prevent that from being cut off.

Reply:
Would like to keep the minimum width of the overall grid to 300px for mobile, but I'll play around with the column widths.

Reply:
Made text columns flexible. If we get more columns included in between them in the future, we could make them flexible as well.

https://github.com/codeforpdx/PASS/assets/14917816/1ab60850-c076-4f2b-95d3-7877a2309e0b